### DISCUSSION TITLE
Add Traffic Insights to Mobile Github Apps

### DISCUSSION THREAD

**Discussion by GitTorre** [community/community#6842] [Mobile]
### Intro

The primary usefulness of the mobile Github app (I use the iOS version), is to quickly interact with your repos (any Issues?, any Pull Requests? new stars?...). 

One of the things that is important to know without having to launch the website in a browser is how many clones have taken place in the last 1 - n days, how many unique visits in the last 1 - n days; basic traffic data.

This information does not need to be presented as it is on the full website (with interactive charts/graphs); rather, just the good old fashioned numeric data (just like Issues at a glance, for example. That is, if you see Issues greater than 0, well, you will click on Issues and address/interact with the issue from the mobile app). 

This information is useful, especially when you just released a new version or a preview version where you are seeking feedback and want to know how many folks are interacting with your repo.

### Introducing the Github Repo Traffic at a Glance Feature 

**User Experience**

#### View link path:    **Favorites** -> **Repo** -> **...More** -> **Traffic**

Clicking on the Traffic link, which would live under ...More to not add clutter to the default experience, would take you to a simple view where you would see clone count and user traffic count for a date range, with the default being one week's worth of data.

You could extend this range to the maximum amount of time Github stores data (like, two weeks for Clones, or whatever it actually is) by clicking on the date header, which would show a drop down control to choose a new supported (e.g., data available) range.

Example View (simple by design, clear and concise):

### Traffic: 10/19 - Today

**Git clones**
```34 Clones by 19 Unique cloners```

**Visitors**
```1,092 Views by 134 Unique visitors```

It is clear that I am not a designer, but the idea is that this new view should be **really** simple. Charts/Graphs are frosting. The cake matters most here.

Thanks for considering this simple, yet useful Traffic feature. It would start the ball rolling on being able to add more Insights in the future, but this is most likely the easiest + useful Insights feature addition to add to your mobile app roadmap. . 

...Charles

**Comment by brianlovin** [comment]
Wow, what a writeup @GitTorre — thank you. I agree, insights and the Traffic subtab would be awesome to support natively in the app, even in a simplified way at first with just plain numbers. I've created a tracking issue for our team to think about this some more, but it's not currently prioritized on our roadmap.

Thank you for taking the time to write up such a detailed spec!

**Reply by dberardo-com** [reply]
Any info on this feature

**Comment by sumonhasan12345** [comment]
> ### Intro
> 
> 
> 
> The primary usefulness of the mobile Github app (I use the iOS version), is to quickly interact with your repos (any Issues?, any Pull Requests? new stars?...). 
> 
> 
> 
> One of the things that is important to know without having to launch the website in a browser is how many clones have taken place in the last 1 - n days, how many unique visits in the last 1 - n days; basic traffic data.
> 
> 
> 
> This information does not need to be presented as it is on the full website (with interactive charts/graphs); rather, just the good old fashioned numeric data (just like Issues at a glance, for example. That is, if you see Issues greater than 0, well, you will click on Issues and address/interact with the issue from the mobile app). 
> 
> 
> 
> This information is useful, especially when you just released a new version or a preview version where you are seeking feedback and want to know how many folks are interacting with your repo.
> 
> 
> 
> ### Introducing the Github Repo Traffic at a Glance Feature 
> 
> 
> 
> **User Experience**
> 
> 
> 
> #### View link path:    **Favorites** -> **Repo** -> **...More** -> **Traffic**
> 
> 
> 
> Clicking on the Traffic link, which would live under ...More to not add clutter to the default experience, would take you to a simple view where you would see clone count and user traffic count for a date range, with the default being one week's worth of data.
> 
> 
> 
> You could extend this range to the maximum amount of time Github stores data (like, two weeks for Clones, or whatever it actually is) by clicking on the date header, which would show a drop down control to choose a new supported (e.g., data available) range.
> 
> 
> 
> Example View (simple by design, clear and concise):
> 
> 
> 
> ### Traffic: 10/19 - Today
> 
> 
> 
> **Git clones**
> 
> ```34 Clones by 19 Unique cloners```
> 
> 
> 
> **Visitors**
> 
> ```1,092 Views by 134 Unique visitors```
> 
> 
> 
> It is clear that I am not a designer, but the idea is that this new view should be **really** simple. Charts/Graphs are frosting. The cake matters most here.
> 
> 
> 
> Thanks for considering this simple, yet useful Traffic feature. It would start the ball rolling on being able to add more Insights in the future, but this is most likely the easiest + useful Insights feature addition to add to your mobile app roadmap. . 
> 
> 
> 
> ...Charles

**Comment by sumonhasan12345** [comment]
Suggest on answer
